sklearn/manifold/__init__.py

```diff
@@ -7,6 +7,8 @@
 from .mds import MDS
 from .spectral_embedding_ import SpectralEmbedding, spectral_embedding
 from .t_sne import TSNE
+from .laplacian_eigenmap_ import LaplacianEigenmap, laplacian_eigenmap
 
 __all__ = ['locally_linear_embedding', 'LocallyLinearEmbedding', 'Isomap',
-           'MDS', 'SpectralEmbedding', 'spectral_embedding', "TSNE"]
+           'MDS', 'SpectralEmbedding', 'spectral_embedding', "TSNE",
```

Comment:
You can remove `SpectralEmbedding` and `spectral_embedding` as they are now deprecated...